.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2
+jobs:
+  build_docs:
+    docker:
+      # see: https://circleci.com/docs/2.0/circleci-images/#python
+      - image: circleci/python:3.7-stretch
+    steps:
+      # Get our data and merge with upstream
+      - run: sudo apt-get update
+      - checkout
+
+      - restore_cache:
+          keys:
+            - cache-pip
+
+      - run: |
+          pip install numpy==1.17.4
+          pip install --user .[docs,testing]
+
+      - save_cache:
+          key: cache-pip
+          paths:
+            - ~/.cache/pip
+
+      # Build the docs
+      - run:
+          name: Build docs to store
+          # nit-picky mode, turn warnings into errors,
+          # but do not stop the build on errors (so we can still inspect the doc artifacts)
+          # this needs to re re-instated when doc-revamp merged
+          # command: |
+          #   sphinx-build -b html -nW --keep-going -d docs/_build/doctrees docs/source docs/_build/html
```

Comment:
Can remove this. I have made a note in the issue that will take care of this before merging `docs-revamp` into `develop`: https://github.com/aiidateam/aiida-core/issues/4050

Reply:
Fixed in 9bebcb7